sql-server.py

```diff
@@ -767,8 +767,8 @@ def safe_arg(request_arg):
 
 
 # Add all REST definitions here
-api.add_resource(Case, '/cases/<case_id>')
-api.add_resource(CaseList, '/cases')
+api.add_resource(Case, '/api/v1/cases/<case_id>')
+api.add_resource(CaseList, '/api/v1/cases')
```

Comment:
Well, from what I've read on APIs it is good form to have the version of your API included either in the url or in the header which is why I added the version. Then if at some point we publish a new version of the spec that breaks backward compatibility we can call it v2 and leave v1 also running.